report/extra_metadata_templates.go

```diff
@@ -0,0 +1,33 @@
+package report
+
+type ExtraMetadataTemplates []MetadataRowTemplate
```

Comment:
> Why "Extra" ...

See "all templates (not just extra, and including metrics) need to come from the probe" todo note above.

> not a map

It's eventually going to be all templates, which means order will matter, unless we added a "priority" to the MetadataRowTemplate (could replace `Prime`, just have everything over priority X displayed by default).

> commutative

That concerns me too. In fact, the extra templates would be added in the order the plugins are called, which is potentially non-deterministic. I was debating adding a "sortable" priority column, which would let us (mostly?) make merge commutative. Any other alternatives?